backend/api_gateway/api_gateway/settings/base.py

```diff
@@ -41,7 +43,12 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'user',
+    'backend.user_service.user',
+    'backend.group_service.group',
```

Comment:
Where 'user_service' and 'groujp_service' is used?

Reply:
Yes, Both are needed.